docs/src/index.md

```diff
@@ -1,21 +1,40 @@
-# DataFrames Documentation Outline
+# DataFrames.jl
+
+Welcome to the DataFrames documentation! This resource aims to teach you everything you need
+to know to get up and running with tabular data manipulation using the DataFrames.jl package
+and the Julia language. If there is something you expect DataFrames to be capable of, but
+cannot figure out how to do, please reach out with questions in Domains/Data on
+[Discourse](https://discourse.julialang.org/new-topic?title=[DataFrames%20Question]:%20&body=%23%20Question:%0A%0A%23%20Dataset%20(if%20applicable):%0A%0A%23%20MWE%20(if%20applicable):%0A&category=Domains/Data&tags=question).
+You can also give feedback and suggest features or improvements by
```

Comment:
"give feedback" is too general, better say "report bugs". Also "suggest features" is likely to attract many little useful posts which we will have to close, creating frustration on both sides. Better leave people discuss their problems on Discourse.